internal/cmd/group_flag.go

```diff
@@ -190,10 +190,46 @@ func countFlags(flags ...string) (count int) {
 }
 
 const MaxAWSDBSizeBytes = 1024 * 1024 * 1024 * 20 // 20 GB
+func checkIfDump(filename string) (bool, error) {
+	file, err := os.Open(filename)
+	if err != nil {
+		return false, err
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	if scanner.Scan() {
+		firstLine := scanner.Text()
+		if strings.TrimSpace(firstLine) == "PRAGMA foreign_keys=OFF;" {
```

Comment:
Why not `return strings.TrimSpace(firstLine) == "PRAGMA foreign_keys=OFF;", nil`?